### DISCUSSION TITLE
Savable emote lists for easy holiday decorating

### DISCUSSION THREAD

**Discussion by ProblyStoned** [night/betterttv#5852] [Ideas]
Something I like to do when the new season starts is change the emotes I have now into the seasonal versions of them, but this can be very time consuming. If we have the ability to save some emote presets, then we could save a few for different holidays and hot swap em when we needed to.

**Comment by susyemeralds** [comment]
Alright so not only should we have the ability to make collections of emotes that are enabled on our channel but also collections of emotes that are not currently enabled or even collections that mixes both of these. 

What would be amazing would be the ability to have a predefined list of emotes that we want to swap out for one another like:
- WAYTOODANK currently enabled on my channel would be swapped for XMASWAYTOODANK
and so forth, going through a whole list of emotes. 

We could have several of these lists at the ready for each holiday or for other occasions!

We could also imagine programming when we want that whole list of emotes to be swapped out for the other list of emotes. We could also then imagine being able to program when they are swapped back to the original list of emotes.
